### PR TITLE
Bugfix/text for related media

### LINKED ISSUE #2370: Organization view: Incorrect user info. text for 'Add related media'
# Steps  
1.Visit https://staging.apinf.io  v 0.42.0 
2.Login as admin 
3.Add organization 
4.Go to above added organization from catalog 

# Expected Result  
 User info. text for "Add Related Media" should be "You don't have related media to this Organization"  

# Actual Result  
User info. text is incorrect  "You don't have related media to this API"     

# Screenshot  
organization_add_related_media_info_text

### PR DESCRIPTION
**Pull request by marla-singer**
Closes #2370

@apinf/developers Ready for review

### CODE REVIEW

File: core/lib/i18n/en.i18n.json [hunk 1]
```diff
@@ -496,7 +496,10 @@
   "relatedMedia_addButton_help": "Add embedded related media content",
   "relatedMedia_addButton": "Add related media",
   "relatedMedia_Title": "Related Media",
-  "relatedMedia_Well_Message": "You don't have any media related to this API",
+  "relatedMedia_Well_Message": {
+      "api": "You don't have any media related to this API",
```
Comment (nnn) [inline core/lib/i18n/en.i18n.json]: Add dot at the end of sentence or remove one from the next line.

File: related_media/client/related_media/media.js [hunk 2]
```diff
@@ -33,20 +39,28 @@ Template.relatedMedia.onCreated(function () {
...
 
     // Return items of organization collection via Pagination
-    return posts;
+    return instance.pagination.getPage();
   },
   templatePagination () {
     // Get reference of pagination
     return Template.instance().pagination;
   },
+  wellMessage () {
```
Comment (nnn) [inline related_media/client/related_media/media.js]: Any better name for this helper? I would suggest `'noRelatedMediaMessage'`.
Don't forget to rename i18n tokens after you change the name.

File: core/lib/i18n/en.i18n.json (hunk 1 above)
Comment (marla-singer) [inline core/lib/i18n/en.i18n.json]: @nnn Actually, don't understand this remark. `remove one from the next line` -- what does it mean?

Comment (nnn) [inline core/lib/i18n/en.i18n.json]: @marla-singer Just a small thing. You have a dot at the end of the sentence on line 501, but you don't have it at the end of the sentence on line 500.

**Comment by marla-singer** [comment]
@nnn Done

File: core/lib/i18n/en.i18n.json (hunk 1 above)
Comment (brylie) [inline core/lib/i18n/en.i18n.json]: Basically, we want to be consistent with punctuation. If we use periods (punctuation) in some i18n strings, we should use them in others.

**Pull request merged**